recipe/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "authzed" %}
+{% set version = "1.20.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7610e6b897d4a61765739183e68ca3de32cbc6dc0dc1df247444ae129e58627d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv
+  skip: True  # [py<38]
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - poetry-core >=1.0.0
+  run:
+    - python
+    - grpcio >=1.63
+    - protobuf >=5.26,<6
+    - grpc-interceptor >=0.15.4
+    - googleapis-common-protos >=1.65.0
+
+test:
+  imports:
+    - authzed.api.v1
+  requires:
+    - pip
+  commands:
+    - pip check
+    - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
+    # Tests depends on missing packages
+
+about:
+  home: https://github.com/authzed/authzed-py/tree/main
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE.txt
```

```suggestion
  license_file: LICENSE
```
This should work for tarball's license